frontend/src/components/marketplace/ControlledCarousel.jsx

```diff
@@ -0,0 +1,40 @@
+import React, { useState } from 'react'
+import Carousel from 'react-bootstrap/Carousel'
+import ItemCard from '../common/ItemCard'
+
+const RECOMMANDATION_STANDARD = 1
+
+function ControlledCarousel (prop) {
+  const [index, setIndex] = useState(0)
+
+  const handleSelect = (selectedIndex, e) => {
+    setIndex(selectedIndex)
+  }
+
+  const items = prop.items
+
+  const itemList = items.map((item) => {
+    if (!item.numberOfTimesBorrowed || item.numberOfTimesBorrowed < RECOMMANDATION_STANDARD) {
+      return null
+    }
```

Comment:
Add a title to this component so we know what it is

Reply:
this is still missing